pom.xml

```diff
@@ -122,6 +126,7 @@
         <module>debezium-scripting</module>
         <module>debezium-server</module>
         <module>debezium-testing</module>
+        <module>debezium-api-generator</module>
```

Comment:
See my comment above. We can easily extract the API generator and move it to a separate repo. This was just handy for the initial development.